### PR TITLE
Add the option to show console logging in the terminal

### PR DESCRIPTION
**Pull request by jcgertig**
preview

### CODE REVIEW

**Comment by nitin42** [comment]
GREAT  WORK !!!!!! @jcgertig . Thank you so much for adding this. I will definitely merge this later because I am currently working on this

File: components/Terminal.css [hunk 1]
```diff
@@ -1,44 +1,41 @@
-.container-main {
+.terminal-container-wrapper {
```
Comment (nitin42) [inline components/Terminal.css]: Thanks for renaming the classes!

File: package.json [hunk 2]
```diff
@@ -29,7 +31,8 @@
   "scripts": {
     "start": "NODE_ENV=development ./node_modules/.bin/webpack-dev-server --watch",
     "build": "NODE_ENV=production ./node_modules/.bin/webpack --config ./webpack.config.js --display-max-modules 0",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "dev": "cross-env NODE_ENV=development ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./test/webpack.config.js --progress --colors --content-base test/"
```
Comment (nitin42) [inline package.json]: I've added a started kit (covers local dev server and webpack config). So this will not be required 😄

File: test/entry.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (nitin42) [inline test/entry.js]: Again! We don't need this!

File: test/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (nitin42) [inline test/index.html]: I've created a webpack config for dev env and it includes the plugins `html-webpack-template` and `html-webpack-plugin`. So we don't need a html file anymore to test the project locally !!

File: test/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+const webpack = require('webpack');
```
Comment (nitin42) [inline test/webpack.config.js]: I've created a new config from scratch. You can make changes in that file.

**Comment by nitin42** [comment]
I've left some comments. Let me know what you think !

**Comment by jcgertig** [comment]
@nitin42 sounds good go ahead and add your starter and html-webpack-plugin and I will deal with merge conflicts

**Comment by jcgertig** [comment]
@nitin42 can I ask what editor you use?

**Comment by nitin42** [comment]
What happened ?

**Comment by jcgertig** [comment]
You use tab characters not spaces that's why I was wondering but I dropped in a editor config so if your editor supports it or has a plugin for it I would suggest using it

**Comment by nitin42** [comment]
Thank you so much @jcgertig ! Could you rebase and create another PR which you created earlier which included:

* Edited css classnames
* console logging option

Because I've merged #5  and have noticed no changes!

Thank you !! 😄

**Comment by nitin42** [comment]
Thank you so much for the PR! 😄

**Pull request merged**